### PR TITLE
[Dev] adds enzyme

### PR DESCRIPTION
**Pull request by l2succes**
adds the enzyme test utility library that makes it easier to write tests. 

Skip New Tests

### CODE REVIEW

File: src/lib/Components/Inbox/Conversations/__tests__/Composer-tests.tsx [hunk 1]
```diff
@@ -24,4 +26,13 @@ describe("regarding the send button", () => {
 
     expect(tree).toMatchSnapshot()
   })
+
+  it("calls onSubmit with the text when send button is pressed", () => {
+    const onSubmit = jest.fn()
```
Comment (l2succes) [inline src/lib/Components/Inbox/Conversations/__tests__/Composer-tests.tsx]: This tests the Composer `onSubmit` event handler gets called and behaves as intended when the send button is pressed

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Home/ArtistRails/__tests__/ArtistRail-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Inbox/Conversations/__tests__/Composer-tests.tsx [hunk 2]
```diff
@@ -24,4 +26,13 @@ describe("regarding the send button", () => {
...
+    const wrapper = shallow(<Composer onSubmit={onSubmit} />)
+    const text = "Don't trust everything you see, even salt looks like sugar"
```
Comment (sarahscott) [inline src/lib/Components/Inbox/Conversations/__tests__/Composer-tests.tsx]: 😂

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Home/ArtistRails/__tests__/ArtistRail-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Inbox/Conversations/__tests__/Composer-tests.tsx (hunk 2 above)
Comment (alloy) [reply src/lib/Components/Inbox/Conversations/__tests__/Composer-tests.tsx]: Hahaha

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Home/ArtistRails/__tests__/ArtistRail-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Home/ArtistRails/__tests__/ArtistRail-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Home/ArtistRails/__tests__/ArtistRail-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by alloy** [comment]
Looks sweet, now let’s write some mad tests!

File: package.json [hunk 3]
```diff
@@ -126,40 +120,23 @@
...
-    "*.@(ts|tsx)": [
-      "tslint --fix",
-      "npm run prettier-write --",
-      "git add"
-    ],
-    "*.json": [
-      "npm run prettier-write --"
-    ]
+    "*.@(ts|tsx)": ["tslint --fix", "npm run prettier-write --", "git add"],
+    "*.json": ["npm run prettier-write --"]
```
Comment (orta) [inline package.json]: ah, this is wy the formatting keeps changing, it needs a `git add` like above

Comment (alloy) [reply package.json]: Ah, hah

**Pull request merged**